app/scrcpy.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=scrcpy
+GenericName=Android Remote Control
+Comment=Display and control your Android device
+Exec=scrcpy
```

Comment:
At first, it did not work for me.

To get the error, I changed to:
```
Exec=bash -c "scrcpy &> /tmp/log"
```
And `/tmp/log` contained:
```
exec: No such file or directory
ERROR: 'adb' command not found (make it accessible from your PATH or define its full path in the ADB environment variable)
ERROR: Could not execute "adb push"
```
In fact, the `adb` directory is added in `PATH` in my `.bashrc`, so it's not in `PATH` when the session is started, so starting it for the menu does not work in that case.

In practice, do you use `android-tools-adb`, or did you set your `adb` in `PATH` from somewhere else?

Reply:
I add the file in `/usr/share/applications/scrcpy.desktop` with content:
```
[Desktop Entry]
Name=scrcpy
GenericName=Android Remote Control
Comment=Display and control your Android device
Exec=scrcpy
Icon=gparted
Terminal=false
Type=Application
Categories=Utility;RemoteAccess;
StartupNotify=false
```
After it, click the icon from menu. Scrcpy start as expected.

Reply:
@rom1v 

Maybe you can try to change Exec to `echo $PATH > /tmp/path_out` and execute again to get more info for debugging.

Reply:
I guess `bash -i scrcpy` will work for `adb` not in standard paths? That bash option forces the interactive mode and `.bashrc` is sourced. I use [android-tools](https://archlinux.org/packages/community/x86_64/android-tools/) on Arch Linux, where adb is installed under /usr/bin, so I cannot test my idea.